brightwind/export/export.py

```diff
@@ -154,3 +154,97 @@ def export_csv(data, file_name=None, folder_path=None, **kwargs):
     else:
         raise NotADirectoryError("The destination folder doesn't seem to exist.")
     print('Export to csv successful.')
+
+
+def export_tws_file(eastings, northings, height, wspd_series, direction_series, wspd_std_series=None, site_name=None, file_name=None, folder_path=None):
+    """Export a WindSim timeseries tws file using a timeseries dataframe of wind speed and direction.
+
+    :param eastings: Eastings of the measurement location in meters.
+    :type eastings: float
+    :param northings: Northings of the measurement location in meters.
+    :type northings: float
+    :param height: Height that the timeseries represents in meters.
+    :type height: float
+    :param wspd_series: Series of wind speed variable
+    :type wspd_series: pandas.Series
+    :param direction_series: Series of wind directions between [0-360]
+    :type direction_series: pandas.Series
+    :param wspd_std_series: Series of wind speed standard deviations
+    :type wspd_std_series: pandas.Series
+    :param site_name: The site name to include in the file.
+           i.e 'Demo Data'
+    :type site_name: str
+    :param file_name: The file name under which the tab file will be saved, or use the default,
+        i.e '2019-06-07_brightwind_tws_export.tab'
+    :type file_name: str
+    :param folder_path: The directory where the tab file will be saved, default is the working directory.
+    :type folder_path: str
+
+    :return: Creates a WindSim timeseries tws file which can be used in the WindSim software.
+
+    **Example Usage**
+    ::
+        import brightwind as bw
+        df = bw.load_campbell_scientific(bw.demo_datasets.demo_campbell_scientific_site_data)
+
+        bw.export_tws_file(df.Spd80mN, df.Dir78mS, 'Demo Data', 80, file_name='campbell_tws_file', 626100, 827971, folder_path=r'C:\some\folder\')"""
```

Comment:
A few comments on this example:
1. Reorder the inputs here to reflect the function.
2. Assign `site_name='Demo Data'`.
3. Remove the folder path as the example will crash if a user runs it because it doesn't exist for them. It will also crash because you've added a trailing `\` which is interpreted as the string not finishing. You can add an example folder path with an `i.e.` above, like you've done before.

Reply:
Updated, but note that point 3 appears to also be incorrect in the export_tab_file() function.